### DISCUSSION TITLE
A Problem encountered when using data parallel

### DISCUSSION THREAD

**Discussion by Aziily** [hpcaitech/ColossalAI#1540] [Community | Q&A]
Hi, I think I may need someone to help me solve the problem
I am trying to use two nodes which each has a GPU, and I have written such config in config.py
![image](https://user-images.githubusercontent.com/101855289/188264814-2ecedb94-815e-485d-9928-28b52de289b3.png)

while when I use colossalai run to start the distributed training, I get such WARNING
![image](https://user-images.githubusercontent.com/101855289/188264872-b0aa8d6f-ff52-46a3-9d39-6b8eb186e718.png)

and by watching the logs of training, I believe it doesn't train a model, instead it starts training a model per process

**Comment by YuliangLiu0306** [comment]
Did you set the config about 'zero'?
